### ISSUE TITLE
paper-material focus state issue

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-material#8]
The focus state is not visible. We need a clearer way to identify which item is in focus.

**Comment by blasten** [comment]
`paper-material` doesn't have a `focus` state. However, you could look at https://github.com/PolymerElements/paper-button that uses `paper-material` and does have a focus state.

**Comment by lpalmaro** [comment]
Awesome - nice and clear. Thanks!
